### ISSUE TITLE
we need to define the most basic feature list

### ISSUE THREAD

**Issue by vesparny** [getPoseidon/Poseidon#11]
- basic user management (login, logout, barebone profile page)
- post CRUD
- default theme

everything should be a plugin
- commenting system (disquss or whatever)

**Comment by scottwolpow** [comment]
I agree. This keeps it very small and you only add what you need. It will make it much easier for non tech people to use. They w ill not see options that just confuse them.
